src/backend/api/Fusion.Resources.Logic/Workflows/AllocationJointVentureWorkflowV1.cs

```diff
@@ -69,6 +69,7 @@ public WorkflowStep SkipApproval()
             switch (current.Id)
             {
                 case CREATED:
```

Comment:
Is this a fix for the workflow or is it regarding auth?

Reply:
It's related to the changes with request router.If there's never a need to approve joint venture requests, we should update the security matrix and revert this.